phpBB/phpbb/posting/post_helper.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ *
+ * This file is part of the phpBB Forum Software package.
+ *
+ * @copyright (c) phpBB Limited <https://www.phpbb.com>
+ * @license GNU General Public License, version 2 (GPL-2.0)
+ *
+ * For full copyright and license information, please see
+ * the docs/CREDITS.txt file.
+ *
+ */
+
+namespace phpbb\posting;
+
+use phpbb\db\driver\driver_interface;
+
+class post_helper
+{
+	/**
+	 * @var driver_interface
+	 */
+	protected $db;
+
+	public function __construct(driver_interface $db)
+	{
+		$this->db = $db;
+	}
+
+	/**
+	 * Get last post id
+	 */
+	public function get_max_post_id(): int
```

Comment:
Where is this being used? Right now this helper seems unused?

Reply:
I am thinking it should be called post_repository.
It is used in https://github.com/phpbb/phpbb/pull/5439/files#diff-61cfe4084cdd26143c1345990d3b2ede58e4305895c0207bf2824bfc91027352R122 for example.
I have added it, because that method can be reused in more parts in the code, the query is duplicated. Also, it will help to have everything related to post table together.